engines/pink/pink.h

```diff
@@ -0,0 +1,158 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ */
+
+#ifndef PINK_PINK_H
+#define PINK_PINK_H
+
+#include "common/random.h"
+
+#include "engines/engine.h"
+
+#include "graphics/wincursor.h"
+
+#include "gui/EventRecorder.h"
+#include "gui/debugger.h"
+
+#include "pink/constants.h"
+#include "pink/director.h"
+#include "pink/file.h"
+#include "pink/utils.h"
+#include "pink/pda_mgr.h"
+
+/*
+ *  This is the namespace of the Pink engine.
+ *
+ *  Status of this engine: In Development
+ *
+ *  Internal name of original engine: OxCart Runtime
+ *
+ *  Games using this engine:
+ *  - The Pink Panther: Passport to Peril
+ *  - The Pink Panther: Hokus Pokus Pink
+ *
+ *  Peril game status:
+ *      Fully playable*
+ *
+ *  Pokus game status:
+ *      Fully Playable*
+ *
+ *  Known bugs:
+ *      Walking sprites don't recalculate position
+ *      PDA is not usable
+ *      missing menu
```

Comment:
These differ from the bugs mentioned in the pull request:
- The first item ("walking sprites...") is not mentioned
- The pull request also mentions that "ActionText is not implemented"

Please sync these two :)